README.md

```diff
@@ -148,6 +148,9 @@ services:
           - ip-500/d  # Limit to 500 requests per day per IP
         
         openapi: /path/to/openapi.yaml  # OpenAPI file for request/response validation
+
+        secrets:
```

Comment:
I don't think secrets is a good name, maybe just omit_headers

Reply:
Fixed